HW tink/currency-tracking/src/main/java/ru/currency_tracking/cur_control/CurrencyController.java

```diff
@@ -0,0 +1,58 @@
+package ru.currency_tracking.cur_control;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ru.currency_tracking.model.Currency;
+import ru.currency_tracking.service.CurrencyService;
+
+import java.util.List;
+
+@RequestMapping("/api")
```

Comment:
Можно прямо сюда унести
```suggestion
@RequestMapping("/api/currencies")
```